### DISCUSSION TITLE
WalletConnect network not supported

### DISCUSSION THREAD

**Discussion by daCandyMan** [WalletConnect/walletconnect-monorepo#4242] [Developer Support]
In the past I was able to connect the Telos EVM wallet with my loopring wallet. But all methods that work before are now not supported.
The prompt I am receiving in Loopring wallet with WalletConnect is:
"Cannot Verify"
"Unknown domain: This domain cannot be verified. Check the request carefully before approving."
Wallet address is correct.
Network is eip155:40, no other optional networks
Then near the bottom there is a prompt: "This network is not supported by WalletConnect".

Now I am locked out from my access.

**Comment by glitch-txs** [comment]
We do support all EVM chains, try contacting Loopring on this issue please

**Reply by daCandyMan** [reply]
Loopring emailed me back stating their wallet should have never connected. I'm waiting for a follow-up email from telos support team.
